From ab10ffa52dec2f2f100561e713f735f4daf00ba4 Mon Sep 17 00:00:00 2001
From: Kevin Strasser <kevin.strasser@intel.com>
Date: Thu, 15 Mar 2018 14:00:01 +0530
Subject: [PATCH 16/27] Disable EXT_pixel_format_float for Android
 CtsViewTestCases

Jira: None.
Test: Android CTS should PASS for below
      android.view.cts.PixelCopyTest#testWideGamutWindowProducerCopyToRGBA16F
Signed-off-by: Kevin Strasser <kevin.strasser@intel.com>
Signed-off-by: Kishore Kadiyala <kishore.kadiyala@intel.com>
---
 src/egl/drivers/dri2/egl_dri2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/egl/drivers/dri2/egl_dri2.c b/src/egl/drivers/dri2/egl_dri2.c
index 3500e43..ef705ee 100644
--- a/src/egl/drivers/dri2/egl_dri2.c
+++ b/src/egl/drivers/dri2/egl_dri2.c
@@ -701,7 +701,7 @@ dri2_setup_screen(_EGLDisplay *disp)
       dri2_renderer_query_integer(dri2_dpy,
                                   __DRI2_RENDERER_HAS_CONTEXT_PRIORITY);
 
-   disp->Extensions.EXT_pixel_format_float = EGL_TRUE;
+   disp->Extensions.EXT_pixel_format_float = EGL_FALSE;
 
    if (dri2_renderer_query_integer(dri2_dpy,
                                    __DRI2_RENDERER_HAS_FRAMEBUFFER_SRGB))
-- 
2.7.4

